packages/components/data-table-manager/src/types.tsx

```diff
@@ -231,14 +231,15 @@ export type TColumnProps<Row extends TRow = TRow> = {
 };
 
 export type TDataTableManagerProps<Row extends TRow = TRow> = {
+  withRowSelection?: boolean;
```

Comment:
![image](https://github.com/user-attachments/assets/d2bdec30-22f0-4a99-be64-c07925f94702)

could you add a short description for our future selves :)

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2938/commits/d4ad05447ab56bad68d7fac6104c085a9943877e

Thank you.